src/PPTX.jl

```diff
@@ -2,6 +2,7 @@ module PPTX
 
 using XMLDict
 using EzXML
+using ZipFile
 using DataStructures
 
 import Tables
```

Comment:
At line 11, we can remove `import Pkg.PlatformEngines: exe7z` ?

Reply:
with `ZipFile` though, you don't need to write files to disk, but would need to rethink a bit more of the structure of the code.

Reply:
let's leave this idea for another version. We could leave this PR open for now, or open an issue instead.